staff/sergiomons/css-layout-simple-cv/styles/main.css

```diff
@@ -0,0 +1,81 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+ }
+ html, body {
+    min-height: 100% !important;
+    height: 100%;
+    margin: 40px;
+    background-color: white;
+}
+header{
+  width:100%;
+  height: 20%;
+  border-bottom: 2px solid black;
+  text-align: center;
+  
+  
+}
```

Comment:
Try to organize your css following an ordered structure,
first the generic selectors by <tag>, then the class and finally the #id, the latter being the most specific.

[Check this, 5 point](https://code.tutsplus.com/tutorials/30-css-best-practices-for-beginners--net-6741)